src/schema/sources.ts

```diff
@@ -367,6 +367,66 @@ export const typeDefs = /* GraphQL */ `
       categoryId: String
     ): SourceConnection!
 
+    """
+    Get all available sources via read replica
+    """
+    sourcesReadReplica(
+      """
+      Paginate after opaque cursor
+      """
+      after: String
+
+      """
+      Paginate first
+      """
+      first: Int
+
+      """
+      Fetch public Squads
+      """
+      filterOpenSquads: Boolean
+
+      """
+      Add filter for featured sources
+      """
+      featured: Boolean
+
+      """
+      Filter by category
+      """
+      categoryId: String
+    ): SourceConnection!
+
+    """
+    Get all available sources from read replica
+    """
+    sourcesReadReplica(
```

Comment:
did you add double definition or is diff just funky?

Reply:
@capJavert Yeah it's just a duplicate of sources to verify on production.
Idea is to merge -> test via postman -> revert this change and make the main one use the RR.

Just wanted to be safe and not break prod on deployment if that makes sense?

Reply:
no i get it but sourcesReadReplica query is defined twice?

Reply:
Oh sorry, missed that somehow.